Dockerfile

```diff
@@ -0,0 +1,39 @@
+# FROM ubuntu-upstart:precise
+# http://phusion.github.io/baseimage-docker/
+FROM phusion/baseimage:0.9.9
+
+# Install pre-pre-requisites.
+RUN apt-get update
```

Comment:
Yeah I know it's a super old PR and we'll have to update it. I came back and edited my comment from 🔢 as it only seems to make sense to chain `RUN` commands based on what Docker does behind the scenes.

I'm still doing a Docker course as we speak at the moment so I'm getting my head around all the different pieces as I'm keen to have a decent opinion about how Docker could work with Chassis and all our extensions. Long weekends FTW! I get to learn things!